Content.Client/GameObjects/Components/PDA/PDAVisualizer.cs

```diff
@@ -1,16 +1,32 @@
 ﻿using Content.Shared.GameObjects.Components.PDA;
+using JetBrains.Annotations;
 using Robust.Client.GameObjects;
 using Robust.Client.Interfaces.GameObjects.Components;
+using Robust.Shared.Interfaces.GameObjects;
 
 namespace Content.Client.GameObjects.Components.PDA
 {
+    [UsedImplicitly]
+    // ReSharper disable once InconsistentNaming
     public class PDAVisualizer : AppearanceVisualizer
     {
 
         private enum PDAVisualLayers
         {
             Base,
-            Flashlight
+            Flashlight,
+            IDLight
+        }
+
+        public override void InitializeEntity(IEntity entity)
+        {
+            base.InitializeEntity(entity);
+            var sprite = entity.GetComponent<ISpriteComponent>();
+
+            sprite.LayerMapSet(PDAVisualLayers.Flashlight, sprite.AddLayerState("light_overlay"));
+            sprite.LayerSetShader(PDAVisualLayers.Flashlight, "unshaded");
+            sprite.LayerMapSet(PDAVisualLayers.IDLight, sprite.AddLayerState("id_overlay"));
+            sprite.LayerSetShader(PDAVisualLayers.IDLight, "unshaded");
```

Comment:
Use `LayerMapReserveBlank` here to reserve the layers before actually using them. Just in case they're absent from a prototype.

Reply:
`LayerMapSet` creates the layer if it doesn't exist, so I thought this wouldn't be an issue?

Reply:
yeah, that should be fine actually